etc/containerpilot.json

```diff
@@ -1,43 +1,43 @@
 {
   "consul": "{{ if .CONSUL_AGENT }}localhost{{ else }}{{ if .CONSUL }}{{ .CONSUL }}{{ else }}consul{{ end }}{{ end }}:8500",
-  "preStart": "/usr/local/bin/reload.sh preStart",
-  "logging": {"level": "DEBUG"},
+  "preStart": "generate-config",
   "services": [
     {
       "name": "nginx",
       "port": 80,
-      "health": "/usr/bin/curl --fail --silent --show-error --output /dev/null http://localhost/nginx-health",
+      "health": "curl -fsSo /dev/null http://localhost/nginx-health",
       "poll": 10,
       "ttl": 25,
       "interfaces": ["eth0"]
     },
     {
       "name": "nginx-public",
       "port": 80,
-      "health": "/usr/bin/curl --fail --silent --show-error --output /dev/null http://localhost/nginx-health",
+      "health": "curl -fsSo /dev/null http://localhost/nginx-health",
       "poll": 10,
       "ttl": 25,
       "interfaces": ["eth1", "eth0"]
     }{{ if .ACME_DOMAIN }},
     {
+      "name": "nginx-ssl",
+      "port": 443,
+      "health": "acme init && curl -kfsSo /dev/null https://localhost/nginx-health",
```

Comment:
If we put `-k` here then we're intentionally bypassing the SSL check. I feel like it'd be really useful for the health check to include verifying SSL. Is it feasible to switch out `localhost` for `{{ .ACME_DOMAIN }}` here so that we can ensure SSL is working in the health check?